shared/js/otSdkDriver.js

```diff
@@ -715,6 +719,45 @@ loop.OTSdkDriver = (function() {
         this._subscriberChannel = channel;
         this._checkDataChannelsAvailable();
       }.bind(this));
+
+      this._setupCursorDataChannel(sdkSubscriberObject);
+    },
+
+    /**
+     * Setups the subscriber cursor data channel that will handle the messages
+     * received with the cursor information about the real stream size and position
+     * of the remote cursor.
+     *
+     * @param {OT.Subscriber} sdkSubscriberObject The subscriber object for the stream.
+     */
+    _setupCursorDataChannel: function(sdkSubscriberObject) {
+      sdkSubscriberObject._.getDataChannel("cursor", {}, function(err, channel) {
```

Comment:
This and `_createCursorDataChannel` duplicate a lot of code.

Couldn't we do something like:

    var dataChannels = ["cursor", "text"];
    dataChannels.forEach(setupCursorDataChannel(channel))

That's only approximate, of course ;-)